lib/BUILD.bazel

```diff
@@ -0,0 +1,26 @@
+load("@npm_bazel_typescript//:defs.bzl", "ts_library")
+
+package(default_visibility = ["//:__pkg__"])
+
+exports_files([
+    "closure/zone_externs.js",
+])
+
+filegroup(
```

Comment:
I don't think we need both the `exports_files` and the `filegroup` to expose the zone_externs.js file